tests/test_check.py

```diff
@@ -411,3 +412,15 @@ def test_check_error_output_typing_io(output: io.StringIO) -> None:
         output=output,
     )
     assert "E0001 (syntax-error)" in output.getvalue()
+
+
+def test_precondition_inline_comment_no_error(caplog) -> None:
+    """Test that no warnings are raised when inline comments are included in the preconditions
+    of a function docstring.
+    """
+    python_ta.check_all(
+        "examples/sample_usage/precondition_inline_comment.py",
```

Comment:
Since this file is really only meant for testing purposes, it should be included under `tests/fixtures`